src/__stories__/icon-button-story.tsx

```diff
@@ -1,31 +1,81 @@
 import * as React from 'react';
 import IconButton from '../icon-button';
 import IconCloseRegular from '../generated/mistica-icons/icon-close-regular';
-import catIcon from './images/cat-icon.png';
+import ResponsiveLayout from '../responsive-layout';
+import Box from '../box';
 
 export default {
     title: 'Components/Buttons/IconButton',
     argTypes: {
         type: {
-            options: ['image url', 'svg component'],
+            options: ['neutral', 'brand', 'danger'],
+            control: {type: 'select'},
+        },
+        variant: {
+            options: ['default', 'soft', 'solid'],
+            control: {type: 'select'},
+        },
+        theme: {
+            options: ['default', 'inverse', 'alternative'],
             control: {type: 'select'},
         },
     },
+    parameters: {fullScreen: true},
 };
 
-const handleOnPress = () => window.alert('pressed!');
+const handleOnPress = () => console.log('pressed!');
```

Comment:
This allows you to see the pressed interaction in the button in a more friendly way (without an alert that pops up).